.gitignore

```diff
@@ -233,3 +233,4 @@ Package
 /src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/bin
 /src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/bin
 /src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.ServiceClientAdapter/bin
+/tools/NetCoreCsProjSync/NetCoreCsProjSync/Properties/launchSettings.json
```

Comment:
do we want to have a single rule for launchsettings.json?

Reply:
Fixed.